include/odp/api/spec/dma.h

```diff
@@ -332,7 +332,7 @@ void odp_dma_pool_param_init(odp_dma_pool_param_t *pool_param);
  * The use of pool name is optional. Unique names are not required. However, odp_pool_lookup()
  * returns only a single matching pool. Use odp_dma_pool_param_init() to initialize pool parameters
  * into their default values. Parameters values must not exceed pool capabilities
- * (@see odp_dma_pool_capability_t)
+ * (odp_dma_pool_capability_t).
```

Comment:
Common pattern seems to be "@see" -> "see", here the word is dropped. What's the exception in the DMA files?

Reply:
The thinking was that when the same type is already mentioned in the text right before the braces, using `see` seems unnecessary.